kubetest2-gce/deployer/build.go

```diff
@@ -34,14 +34,17 @@ func (d *deployer) Build() error {
 
 	if d.LegacyMode {
 		// this supports the kubernetes/kubernetes build
-		klog.V(2).Info("starting the legacy build")
-
-		cmd := exec.Command("make", "bazel-release")
-		exec.InheritOutput(cmd)
-		cmd.SetDir(d.RepoRoot)
-		err := cmd.Run()
+		klog.V(2).Info("starting the legacy k/k build")
+		version, err := d.BuildOptions.Build()
 		if err != nil {
-			return fmt.Errorf("error during make step of build: %s", err)
+			return err
+		}
+		// append the kubetest2 run id
+		version += "+" + d.commonOptions.RunID()
+		if d.BuildOptions.CommonBuildOptions.StageLocation != "" {
```

Comment:
nit
```suggestion
		version += "+" + d.commonOptions.RunID()
		// stage build if requested
		if d.BuildOptions.CommonBuildOptions.StageLocation != "" {
```

I'm a little curious why stage needs to be done here, seems like a side effect unrelated to building, but won't block

Reply:
yeah stage is optional here for gce, it's not needed if running build and up together. but needed if we want to build and then up a cluster later.